homeassistant/components/apcupsd/__init__.py

```diff
@@ -20,7 +20,7 @@
 DOMAIN: Final = "apcupsd"
 VALUE_ONLINE: Final = 8
 PLATFORMS: Final = (Platform.BINARY_SENSOR, Platform.SENSOR)
-MIN_TIME_BETWEEN_UPDATES: Final = timedelta(seconds=60)
+MIN_TIME_BETWEEN_UPDATES: Final = timedelta(seconds=2)
```

Comment:
We normally don't allow shorter update interval than five seconds for polling integrations.